kroxylicious-kms-provider-aws-kms/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.kroxylicious</groupId>
+        <artifactId>kroxylicious-parent</artifactId>
+        <version>0.6.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>kroxylicious-kms-provider-aws-kms</artifactId>
+    <version>0.6.0-SNAPSHOT</version>
+
+    <name>AWS KMS</name>
+    <description>A KMS provider backed by a remote instance of AWS KMS</description>
```

Comment:
Is it worth mentioning it does so without using the AWS SDK?

Reply:
I added it to the Javadoc for the io.kroxylicious.kms.provider.aws.kms.AwsKms.